### DISCUSSION TITLE
输入汉字“佺”，模型读取却识别为“全场”或“伽”等其它字符，就是不识别

### DISCUSSION THREAD

**Discussion by Bounger2** [zai-org/ChatGLM3#1018] [Q&A] [answered]
{'messages': [ChatMessage(role='user', content='介绍下佺这个字的发音', name=None, function_call=None), ChatMessage(role='assistant', content='\n 全景 (quán jǐng) is a Chinese word that means "overview" or "total view". It is written with the characters 全 (quán) meaning "total" or "completely", and 景 (jǐng) meaning "scenery" or "sight". The pronunciation of quán jǐng is usually represented as "quán jǐng" in Mandarin Chinese, although some speakers may pronounce it as "quán gǒng".', name=None, function_call=None), ChatMessage(role='user', content='我不太清楚问你什么问题，请帮我生成 3 个问题，引导我继续提问。问题的长度应小于20个字符，按 JSON 格式返回: ["问题1", "问题2", "问题3"]', name=None, function_call=None)], 'temperature': 0.1, 'top_p': 0.8, 'max_tokens': 200, 'echo': False, 'stream': False, 'repetition_penalty': 1.1, 'tools': None}
2024-03-23 13:24:31.317 | DEBUG    | __main__:create_chat_completion:318 - ==== message ====
role='assistant' content='["什麼是全景的發音？", "全景的發音如何正確讀取？", "全景的發音有哪些常見的讀法？"]' name=None function_call=None

**Comment by Bounger2** [comment]
输入“佺”，但是模型不能正确识别这个汉字，而是识别为其他字符

**Comment by zRzRzRzRzRzRzR** [comment, marked as answer]
估计词表没有
